api/tests/acceptance/application/security-pre-handlers_test.js

```diff
@@ -71,7 +71,7 @@ describe('Acceptance | Application | SecurityPreHandlers', () => {
       // given
       const options = {
         method: 'GET',
-        url: '/api/users/1/memberships',
+        url: '/api/users/1/is-certifiable',
```

Comment:
**Question bloquante**

Ce n'est pas plutôt `/api/users/me/memberships` ?

Reply:
Non, vu que on n'appelle plus le preHandler (tu suis pas, Thierry !)

Reply:
Ok je viens de remonter le code pour comprendre.

On veut juste utiliser une URL qui vérifie que l'utilisateur qui fait la demande est bien celui qui est authentifié. 
Donc c'est arbitraire de prendre `/api/users/1/is-certifiable`, mais ça marche puisque l'autre route concernée par la PR ne prend plus d'ID.

Reply:
J'avais pas vu ton message entre temps 😅